spec/facets/DatasetTypeDatasetFacet.json

```diff
@@ -0,0 +1,35 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://openlineage.io/spec/facets/1-0-0/DatasetTypeDatasetFacet.json",
+  "$defs": {
+    "DatasetTypeDatasetFacet": {
+      "allOf": [
+        {
+          "$ref": "https://openlineage.io/spec/2-0-2/OpenLineage.json#/$defs/DatasetFacet"
+        },
+        {
+          "type": "object",
+          "properties": {
+            "datasetType": {
+              "description": "Dataset type, for example: FILE|STREAM|MODEL|TABLE|VIEW.",
+              "type": "string",
+              "example": "FILE"
+            },
+            "subType": {
+              "description": "Optional sub-type within the dataset type (e.g., CSV under FILE).",
```

Comment:
Examples provided suit more dataset format facet than dataset type. Not sure if `subType` is really needed.

Reply:
format sounds like a good idea, But would that overlap with StorageDatasetFacet

Reply:
@ssanthanam185 yeah. I just wanted to point out that `csv` and `file` are not good examples for `subType` field.

Reply:
I changed the example usage of this field in the description. It should fit more than extension of the file.